cmake/TestLib.cmake

```diff
@@ -0,0 +1,52 @@
+################################################################################
+#    Copyright (C) 2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH    #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+include_guard(GLOBAL)
+
+function(run_module_tests)
+  cmake_parse_arguments(PARSE_ARGV 0 ARGS "" "MODULE;PATH" "")
+
+  if(ARGS_PATH)
+    list(APPEND path "${ARGS_PATH}")
+  else()
+    list(APPEND path "${CMAKE_SOURCE_DIR}/src/modules")
```

Comment:
You're using things like `head -20 "${path}/${ARGS_MODULE}.cmake"` below.

So please use `set()` here.

Reply:
right, done